### ISSUE TITLE
sam init: "Error: Multiple lambda image application templates found"

### ISSUE THREAD

**Issue by justinmk3** [aws/aws-sam-cli-app-templates#95]
This started happening in [our CI](https://github.com/aws/aws-toolkit-vscode/commit/274a59303de770db65a86f23b0907bcf10976558) today. Maybe caused by https://github.com/aws/aws-sam-cli-app-templates/commit/27dce06faabe6a05ba791cc9ce2f28f02fe84700 ?

```
$ sam init --name testProject --no-interactive --dependency-manager pip --package-type Image --base-image amazon/python3.8-base

Cloning app templates from https://github.com/aws/aws-sam-cli-app-templates
Error: Multiple lambda image application templates found. This should not be possible, please raise an issue.
```

Note the above command uses `python3.8-base` ;  this does *not* happen with e.g. `python3.7-base` :
    
    $ sam init --name testProject --no-interactive --dependency-manager pip --package-type Image --base-image amazon/python3.7-base

**Comment by justinmk3** [comment]
@aahung @satmihir

**Comment by CoshUS** [comment]
The PR #86 added more app templates to the repo, which makes init not defaulting to the only option available.
Adding `--app-template hello-world` to the init command will address this issue.

**Comment by KFayal** [comment]
Just experienced the same issue when trying to select python3.8

**Comment by justinmk3** [comment]
For reference: cannot use `--app-template` to workaround this, because `sam` does not allow it with `--base-image`:

```
Error:
You must not provide both the --base-image and --app-template parameters.
```

**Comment by justinmk3** [comment]
from https://github.com/aws/aws-sam-cli/issues/2846#issuecomment-854154902 : 

> The issue is fixed in V.1.24.0
> you can now use the `--app-template` argument to resolve this. For example:
> 
> ```
> sam init --name foo --package-type Image --base-image amazon/python3.8-base --app-template hello-world-lambda-im
> ```

**Comment by jfuss** [comment]
This has been fixed in 1.24 of SAM CLI. Closing.
